### ISSUE TITLE
Default timezone should be set to use mktime safely

### ISSUE THREAD

**Issue by GoogleCodeExporter** [patel22p/mytinytodo#29]
```
Since PHP 5.1.0 timezone should be set to use mktime. It throw an exception 
otherwise. 
I'm running mytinytodo on PHP 5.3.1 and I've added :

date_default_timezone_set('Europe/Paris');

on line 12 of init.php to fix the problem. Timezone should be added to 
config.php and 
then set with :

date_default_timezone_set(Config::get('timezone'));

in init.php

I can send a patch if needed.
```

Original issue reported on code.google.com by `Raphael.Emourgeon` on 6 Mar 2010 at 10:09

**Comment by GoogleCodeExporter** [comment]
```
Will be fixed in v1.4.
```

Original comment by `maxpozd...@gmail.com` on 4 Oct 2010 at 7:06
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `maxpozd...@gmail.com` on 25 Oct 2010 at 6:49
- Changed state: **Fixed**
